src/components/popup/home/BuyButton.tsx

```diff
@@ -1,18 +1,34 @@
+import { useHistory } from "~utils/hash_router";
 import { Button } from "@arconnect/components";
 import browser from "webextension-polyfill";
 import styled from "styled-components";
 import arLogoDark from "url:/assets/ar/logo_dark.png";
 
 interface ButtonWrapperProps {
   padding?: boolean;
+  route: string;
+  logo?: boolean;
 }
 
-export default function BuyButton({ padding }: ButtonWrapperProps) {
+export default function BuyButton({
+  padding,
+  route,
+  logo
+}: ButtonWrapperProps) {
+  const [push] = useHistory();
+
+  const targetRoute = route === "/purchase" ? "/purchase" : "/confirm-purchase";
+
   return (
-    <ButtonWrapper padding={padding}>
-      <CustomButton className="normal-font-weight" small fullWidth>
+    <ButtonWrapper padding={padding} route={route} logo={logo}>
```

Comment:
does `logo={logo}` do anything here?

Reply:
the logo prop controls whether we show the arweave logo next to buy now or not @nicholaswma